commons/src/main/java/info/metadude/android/eventfahrplan/commons/extensions/StandardExtensions.kt

```diff
@@ -0,0 +1,16 @@
+@file:JvmName("StandardExtensions")
+
+package info.metadude.android.eventfahrplan.commons.extensions
+
+/**
+ * Calls the specified function [block] with `this` value as its argument if `this` is `false`
+ * and returns `this` value.
+ *
+ * Also see [also].
+ */
+inline fun <Boolean> Boolean.onFalse(block: (Boolean) -> Unit): Boolean {
```

Comment:
To me this feels like taking extension functions too far.

Reply:
I believe it could be a valid use case for methods returning a boolean where one wants to log the error in the `false` case. I try to avoid have the if-false check over and over.

Reply:
:heavy_check_mark: Renamed in a4f7c3a1d247992e2f684f6b1a1c7885d4c49a1e. Thanks.